requirements-test.txt

```diff
@@ -1,9 +1,9 @@
 # Supported models
-scikit-learn==0.22.2.post1
+scikit-learn==0.23.0
```

Comment:
I think that as scikit-learn doesn't provide Python 3.5 wheels anymore and Python 3.5 blocks us from better performance (#209) we can drop Python 3.5 support right now.
https://pypi.org/project/scikit-learn/#files

Reply:
Another one reason to drop support of Python 3.5:
```
/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/xgboost/__init__.py:28: FutureWarning: Python 3.5 support is deprecated; XGBoost will require Python 3.6+ in the near future. Consider upgrading to Python 3.6+.
  FutureWarning)
```